.vscode/settings.json

```diff
@@ -4,5 +4,5 @@
         "GITLAB_TOKEN": "ACCTEST1234567890123",
         "GITLAB_BASE_URL": "http://127.0.0.1:8080"
     },
-    "go.testFlags": ["-count=1", "-v"]
```

Comment:
Do you think we want to add this to the settings.json? It seems like that may be confusing if someone clicks play on a unit test to run it, and it passes because it's skipped?

Reply:
I didn't put build tags on the unit test files, so they still run when you use `acceptance`.

Reply:
Ah, nice. I assumed that it would only run files explicitly tagged.